### ISSUE TITLE
Can not use in Jupyter on VPS -- No module named tkinter

### ISSUE THREAD

**Issue by NiklasRosenstein** [astropy/astroquery#955]
I'm using Jupyter on my VPS. Python on this server does not have the _tkinter module, thus I can't import astroquery. Can this be fixed? Maybe tolerate tkinter not being available.

**Comment by keflavich** [comment]
Could you post the full traceback here?  Astroquery doesn't require tkinter, so it must be a requirement of something astroquery relies on.

**Comment by NiklasRosenstein** [comment]
I forgot to mention that I was trying to use the `astroquery.gaia` module.

```
Python 3.4.2 (default, Oct  8 2014, 10:45:20)
[GCC 4.9.1] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import astroquery
>>> import astroquery.gaia
Traceback (most recent call last):
  File "/usr/lib/python3.4/tkinter/__init__.py", line 39, in <module>
    import _tkinter
ImportError: No module named '_tkinter'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/niklas/jupyter/lib/python3.4/site-packages/astroquery/gaia/__init__.py", line 18, in <module>
    from astroquery.utils.tap.core import TapPlus
  File "/home/niklas/jupyter/lib/python3.4/site-packages/astroquery/utils/tap/__init__.py", line 18, in <module>
    from astroquery.utils.tap.core import Tap
  File "/home/niklas/jupyter/lib/python3.4/site-packages/astroquery/utils/tap/core.py", line 22, in <module>
    from astroquery.utils.tap.gui.login import LoginDialog
  File "/home/niklas/jupyter/lib/python3.4/site-packages/astroquery/utils/tap/gui/login.py", line 20, in <module>
    from six.moves.tkinter import Tk as TKTk
  File "/home/niklas/jupyter/lib/python3.4/site-packages/six.py", line 203, in load_module
    mod = mod._resolve()
  File "/home/niklas/jupyter/lib/python3.4/site-packages/six.py", line 115, in _resolve
    return _import_module(self.mod)
  File "/home/niklas/jupyter/lib/python3.4/site-packages/six.py", line 82, in _import_module
    __import__(name)
  File "/usr/lib/python3.4/tkinter/__init__.py", line 41, in <module>
    raise ImportError(str(msg) + ', please install the python3-tk package')
ImportError: No module named '_tkinter', please install the python3-tk package
```

**Comment by keflavich** [comment]
Oh.  It looks like the GAIA module actually does use tkinter.  I think there's actually no alternative implemented here.  @jcsegovia, do you have any recommendations on how to work around this, e.g., how to log in without using the gui interface?

**Comment by NiklasRosenstein** [comment]
It's possible to use the `Gaia.login()` method instead of `Gaia.login_gui()`. But it always imports tkinter before I could even decide whether to use the GUI or programatical login. :)

**Comment by jcsegovia** [comment]
I am sorry, but I think that tkinter is mandatory for this module in this
version.

I am not familiar with VPS but, it would be possible to install tkinter
python module in that environment (even if GUI capabilities are not going
to be used), so the module can be imported successfully and then
Gaia.login(usr/pwd) or Gaia.login(file) can be called...?

In the next version I will catch tkinter imports in order to avoid import
errors.

**Comment by bsipocz** [comment]
Fixed by #983
